### ISSUE TITLE
Nothing but white screen

### ISSUE THREAD

**Issue by KenSharp** [ValveSoftware/steam-for-linux#9851]
#### Your system information

* Steam client version (build number or date): 1.0.0.78 (according to APT)
* Distribution (e.g. Ubuntu): Ubuntu 22.02
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: 
[Console log](https://github.com/ValveSoftware/steam-for-linux/files/12051659/steam.log)
[Screen capture](https://github.com/ValveSoftware/steam-for-linux/assets/13607576/7352784f-8ab9-4b07-9c89-5db6d69c9193)
[APT history](https://github.com/ValveSoftware/steam-for-linux/files/12051683/history.log)
Note that ~/.steam/logs does not exist on my system.
* GPU: Nvidia

I've used Steam for years without issue. Yesterday I loaded Steam as normal, but nothing works. I either get an all white screen, or a white screen with some black blocks. Either way Steam is now unusable.

I don't know where to start to debug. I have attached my APT history log in case it's an updated library or driver that could have caused the problem.

It doesn't look like the associated snaps (linux-steam-integration and solus-runtime-gaming) have been updated recently.

#### Steps for reproducing this issue:

1. Open Steam

**Comment by KenSharp** [comment]
Tried `-vgui` and got this instead:

![vgui](https://github.com/ValveSoftware/steam-for-linux/assets/13607576/d7359782-4ecb-4d50-b825-3cef1e3484d5)

**Comment by KenSharp** [comment]
Found the logs.
[logs.zip](https://github.com/ValveSoftware/steam-for-linux/files/12052187/logs.zip)

Tried nuking the Steam directory (~/.local/share/Steam) but it made no difference.

HOWEVER:
```
! Controller 0 attributes:
  Type: 33
  ProductID: 1
  Serial: Awaiting_Logon
  Capabilities: 001041ff
  Firmware Version: 0
  Firmware Build Time: 2147483647 (Tue, 19 Jan 2038 03:14:07 GMT)
  Bootloader Build Time: 2147483647 (Tue, 19 Jan 2038 03:14:07 GMT)
Loaded Config for Local Selection Path for App ID 413090, Controller 0: controller_base/bigpicture_gamepad.vdf
Loaded Config for Local Selection Path for App ID 413090, Controller 0: controller_base/bigpicture_gamepad.vdf
Loaded Config for Local Selection Path for App ID 769, Controller 0: controller_base/basicui_gamepad.vdf
Loaded Config for Last Resort Path for App ID 769, Controller 0: /home/ken/.local/share/Steam//controller_base/basicui_gamepad.vdf
Loaded Config for Local Selection Path for App ID 413090, Controller 0: controller_base/bigpicture_gamepad.vdf
```
After nuking the Steam directory (necessary) **AND** unplugging my gamepad **AND** using `-vgui` I finally got back to a working log-in window. But after logging back in I'm back where I started. Without this process the log-in screen won't even work. Also `-nofriendsui` is completely ignored and the friends window shows a broken image icon.

All in all it's a disaster.

```
X Error of failed request:  BadWindow (invalid Window parameter)
Major opcode of failed request:  12 (X_ConfigureWindow)
Resource id in failed request:  0x6000004
Serial number of failed request:  7061
xerror_handler: X failed, continuing
```

**Comment by literal-0x4e** [comment]
I have this same issue for a couple of weeks now, but in my case it is on Gentoo with AMD Vega64 graphics card, on Mesa, **thus I believe this is not nvidia-specific**.
My client was on beta, so I cleared the beta via command line to get the "stable" client, but that didn't help.
Turning on steam runtime (I otherwise run with it disabled) didn't help either.

**Comment by KenSharp** [comment]
Received an update yesterday and now Steam is completely unusable. Utterly useless.

Still getting the white screen.
![steam](https://github.com/ValveSoftware/steam-for-linux/assets/13607576/f4c80ec3-b6c7-4ccb-b8a9-51a29895e71f)

But now I get another white block which will not go away!
![steam2](https://github.com/ValveSoftware/steam-for-linux/assets/13607576/130d51a9-1730-47e4-a690-c82c2b554e3b)

It appears on top of ALL apps.
![steam3](https://github.com/ValveSoftware/steam-for-linux/assets/13607576/dc67fe57-bd1a-4453-9272-71f6c1111469)

I can't load anything through Steam! I would love to play games I've paid for!

**Comment by literal-0x4e** [comment]
I found a **workaround** for this issue:

1. Shut down steam
2. run
`Steam -forcesteamupdate -forcepackagedownload -overridepackageurl  https://web.archive.org/web/20230531115543if_/http://media.steampowered.com/client -exitsteam`
(this will force steam to downgrade to a May 31st version, fetching the installer package from the web archive)
3. Create a file named `steam.cfg` in the steam directory with these lines inside the file (this will prevent steam from updating itself)
```
BootStrapperInhibitAll=Enable
BootStrapperForceSelfUpdate=False
```

So basically, the workaround is to use an older version of steam until this is fixed.

**Comment by aocard** [comment]
Has there been a development on this? I've been using this workaround for a couple of months but the "settings" menu for specific games is still unusably white, and now I'm starting to get server-side problems about my Steam client being out of date. (Arch Linux, latest proprietary Nvidia drivers)

**Comment by aocard** [comment]
Given that this issue breaks both clicking "Properties..." for a given game, as well as Big Picture and probably other things as well, I thought I'd post a workaround I found for accessing game settings.

The client version from May 31st, much to my happiness, still supports the flag "-oldbigpicture", which allows Big Picture to work without the white glitch breaking the interface. From there it's possible to access settings that aren't available in the normal interface due to this issue, including individual game properties.

**Comment by KenSharp** [comment]
The workaround has another issue.

![AIDS](https://github.com/ValveSoftware/steam-for-linux/assets/13607576/8f104368-b7f5-4060-a27b-8708cdf7c754)

**Comment by KenSharp** [comment]
Re-enabled updates, and tested with a clean directory, and in both cases this now works as intended. Excellent stuff.
